executor_manager/routers/routers.py

```
@@ -394,6 +394,232 @@ async def get_executor_load(http_request: Request):
+            )
+
+        result = executor.get_container_address(executor_name)
+        logger.info(
+            "Resolved executor address: executor_name=%s result=%s",
+            executor_name,
+            result,
+        )
+        return result
+    except HTTPException:
+        raise
+    except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)